src/main/java/duke/command/DeleteCommand.java

```diff
@@ -0,0 +1,38 @@
+/**
+ * Project Duke CS2103
+ * Done by Hong Jin.
+ */
+package duke.command;
+
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+
+/**
+ * public class DeleteCommand that handles delete command that deletes task at index.
+ */
+public class DeleteCommand extends Command{
+    private int index;
+
+    /**
+     * public constructor for DeleteCommand.
+     * @param index for task to delete.
+     */
+    public DeleteCommand(int index) {
+        super();
+        this.index = index;
+    }
+
+    /**
+     * public method execute to execute command.
+     * @param tasks
+     * @param ui
+     * @param storage
+     */
+    @Override
+    public String execute(TaskList tasks, Ui ui, Storage storage) {
```

Comment:
Good idea to have subclasses of Command have an execute method. Ensures consistency across all subclasses and improves readability